youths/models.py

```diff
@@ -50,17 +32,27 @@ class YouthProfile(models.Model):
     )
     volunteer_info = models.TextField(blank=True)
     gender = models.CharField(max_length=32, choices=GENDERS, blank=True)
-    illnesses = ChoiceArrayField(
-        base_field=models.CharField(max_length=32, choices=ILLNESSES),
-        size=4,
-        null=True,
-        blank=True,
-    )
+    diabetes = models.BooleanField(default=False)
+    epilepsy = models.BooleanField(default=False)
+    heart_disease = models.BooleanField(default=False)
+    serious_allergies = models.TextField(blank=True)
+    extra_illnesses_info = models.TextField(blank=True)
     allergies = models.TextField(blank=True)
     notes = models.TextField(
         blank=True
     )  # For documenting e.g. restrictions on gaming or premises
 
+    # Permissions
+    approved_by = models.ForeignKey(
+        Profile,
+        related_name="approved_youth_profiles",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+    )
+    approved_time = models.DateTimeField(null=True, blank=True)
```

Comment:
we probably want to disable the input for this field. you could set `editable=False`, and then mark the field as readonly in `admin.py`: `readonly_fields = ("approved_time",)`

Reply:
Good point. Updated.